### DISCUSSION TITLE
Video Support in Python

### DISCUSSION THREAD

**Discussion by benjaminrwilson** [datoviz/datoviz#24] [Q&A]
Hi @rossant,

What is required to enable video support here? https://github.com/datoviz/datoviz/blob/b01231b67ffc9568e7529e692c6cb7939caa31f2/bindings/cython/datoviz/pydatoviz.pyx#L690 I've tried passing in a path (e.g., `my-video.mp4`), but I seem get a segfault each time.

Additionally, is there a way to pass in a pattern / multiple screenshot strings to the `screenshot` argument of run?

**Comment by rossant** [comment]
What is your OS? Has your version of datoviz been compiled with ffmpeg support? Could you `export DVZ_LOG_LEVEL=0` and copy the console output?

I'm willing to support several ways of saving videos/multiple screenshots in an efficient way. An advantage of Vulkan and datoviz is supporting relatively efficient transfer of frames from the GPU to the CPU in real time.

**Reply by benjaminrwilson** [reply]
Good afternoon @rossant,

I'm using MacOS 10.15.7. It has been compiled with `ffmpeg`. Here's the requested log: 

[log.txt](https://github.com/datoviz/datoviz/files/6680955/log.txt)

**Reply by rossant** [reply]
Thanks. I've had some issues recently with this too. I'll refactor this part soon.

**Comment by rossant** [comment]
There is now a video example in the gallery, using imageio to create the video frame by frame.
